src/CameraScreen.tsx

```diff
@@ -44,6 +44,7 @@ export type Props = {
   frameColor: any,
   torchOnImage: any,
   torchOffImage: any,
+  saveToCameraRoll: boolean,
```

Comment:
Can you make this optional and provide and perhaps default value?
```suggestion
  saveToCameraRoll?: boolean,
```